docs/website/blog/2023-01-24-odo-v3.6.0.md

```diff
@@ -0,0 +1,156 @@
+---
+title: odo v3.6.0 Released
+author: valaparthvi
+author_url: https://github.com/valaparthvi
+author_image_url: https://github.com/valaparthvi.png
+tags: ["release"]
+slug: odo-v3.6.0
+---
+
+odo `v3.6.0` is now out!
+
+<!--truncate-->
+
+To install `odo`, follow [the installation guide](../docs/overview/installation).
+
+## Notable Changes
+Check this Playlist for an overview of the most notable changes in this release: [odo v3.6.0 (Sprint 230)](https://www.youtube.com/playlist?list=PLGMB2PY4SNOrs_t1STsiE9iy_pw1iky0X)
+
+### Features
+
+#### Extend pod and container specification with Devfile attributes `pod-overrides` and `container-overrides`
+It is now possible to extend a pod and its container specification by using `pod-overrides` and `container-overrides` Devfile attributes.
+
+This feature allows you to modify pod specifications such as Service Account, Scheduler Name, Security Context, etc and container specifications such as Security Context, and Resources. However, it restricts from modifying pod specifications such as Init Containers, Containers, and Volumes; and container specifications such as Volume Mounts, Image, Ports, Name, Env, Args, and Commands.
+
+`pod-overrides` and `container-overrides` can be specified at both Component and Devfile level.
+
+Example -
+```yaml
+attributes:
+  pod-overrides:
+    spec:
+      serviceAccountName: my-new-service-account
+  container-overrides:
+    spec:
+      securityContext:
+        runAsUser: 1001
+        runAsGroup: 1001
+```
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/7hMm27MYD94" title="Extend pod and container specification with Devfile attributes" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+#### `odo delete component` now has a new `--running-in` flag
+`--running-in` flag can be used with `odo delete component` to specify the mode in which resources should be deleted. There are 2 modes to choose from "dev", and "deploy". By default, the commands deletes all the resources regardless of the mode.
+
+To delete resources running in 'Dev' mode, i.e. resources created by `odo dev` use `--running-in=dev`.
+
+To delete resources running in 'Deploy' mode, i.e. resources created by `odo deploy` use `--running-in=deploy`.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/WkRc3F46X_0" title="odo delete component --running-in" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+#### `odo` can now be used inside DevSpaces
+Port forwarding done by `odo dev` now uses port numbers starting from 20000 instead of 40000. This change was made to make it possible to run odo inside DevSpaces.
```

Comment:
```suggestion
Port forwarding done by `odo dev` now uses port numbers starting from 20001 instead of 40001. This change was made to make it possible to run odo inside [DevSpaces](https://developers.redhat.com/products/openshift-dev-spaces/overview).
```